index.js

```diff
@@ -37,8 +38,36 @@ module.exports = (filePath, options) => {
 	if (optionsObj.verbose) {
 		console.log(chalk.gray(cmd));
 	}
-
+	var diff = [];
 	const resultBuffer = execSync(cmd);
+	const other_cmd = "grep -inRw -E 'TODO' . --exclude-dir={node_modules,test}"
+	const foo = function(cb){
+		exec(other_cmd, (err, stdout, stderr) => {
+			if (err) {
+				console.error(`exec error: ${err}`);
+				return cb(err);
+			}
+			cb(null, {stdout,stderr});
+		});
+	}
+
+	foo(function(err, {stdout,stderr}) {
```

Comment:
I'll talk to you about this IRL, but I'm wondering if this would make more sense using a git log.
So leverage the existing files that come back from the Todo counter, then do a git history on each of them to dig up when the todo was added. My guess is your only added complexity here would be finding the commands that will do the things.